ci/install-build-deps.sh

```diff
@@ -5,26 +5,24 @@
 # WARNING: This script is destructive to your machine's environment and globally-installed files. For example, the Ubuntu-specific parts of the script
 # modify the contents of /etc/apt. The script is intended to be run inside a container of the corresponding OS, not directly on your machine.
 
-
-OS="$(. /etc/os-release; echo "$ID:$VERSION_ID")"
-
+OS="$(. /etc/os-release; if [ -z ${PLATFORM_ID+x} ]; then echo "$ID:$VERSION_ID"; else echo "$PLATFORM_ID"; fi)"
```

Comment:
```sh
OS="$(. /etc/os-release; echo "${$PLATFORM_ID:-$ID:$VERSION_ID}")"
```

Reply:
This?  `OS="$(. /etc/os-release; echo "${PLATFORM_ID:=$ID:$VERSION_ID}")"`

I was getting an error otherwise.

Reply:
Without that `$` at the front, yes. With `=` instead of `-`, no. Fixed my comment.

Reply:
Fixed